### ISSUE TITLE
Existing code for conversion of mmr to vmr?

### ISSUE THREAD

**Issue by jgliss** [metno/pyaerocom#271]
@hansbrenna, @ejgal For the CAMS84 project I need a helper method that converts mmr (units kg kg-1) to vmr (mol mol-1) that I want to use in ReadGridded.AUX_FUNS to convert mmro3 and mmrno2 to corresponding vmrs. I think I saw something like that in one of your branches @hansbrenna, so I wonder if I should base on already existing code from you, so we do not too many things in parallel.

In general, we probably need some deeper logic (recursive computation and possibility to specify different options for computation of auxiliary model variables in ReadGridded)... but that is another issue =)

**Comment by hansbrenna** [comment]
There's code in the branch dev_gridded_conc_from_vmr in `aux_read_cubes.py`.

**Comment by jgliss** [comment]
Thanks @hansbrenna. The best way to incorporate this would be probably to merge this branch *dev_gridded_conc_from_vmr* into *main-dev*. But I guess this also destroys the conversion of mmr & rho to conc... ?
